webapp/components/DeleteData/DeleteData.spec.js

```diff
@@ -45,7 +43,7 @@ describe('DeleteData.vue', () => {
     }
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-05-27T08:54:36Z" title="Wednesday, May 27th 2020, 10:54:36 am +02:00">May 27, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -68,11 +68,11 @@ describe('DeleteData.vue', () => {
       jest.clearAllMocks()
     })
 
-    it('defaults to deleteContributions to false', () => {
+    it('defaults to deleteContributions to true', () => {
       expect(wrapper.vm.deleteContributions).toEqual(false)
```


Why this is changed to `true` while on the next line it is still `false`?
And the default seems to be `false` if I test it on the platform.

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-07-30T11:39:56Z" title="Thursday, July 30th 2020, 1:39:56 pm +02:00">Jul 30, 2020</time>_

---


@Tirokk now that everything is tested cleanly the question is if this is still up to date ? i have to come back in. :) is default 'true'